egs/wsj/s5/utils/split_data.sh

```diff
@@ -16,20 +16,28 @@
 # limitations under the License.
 
 split_per_spk=true
+split_per_reco=false
 if [ "$1" == "--per-utt" ]; then
   split_per_spk=false
   shift
+elif [ "$1" == "--per-reco" ]; then
+  split_per_spk=false
+  split_per_reco=true
+  shift
 fi
 
 if [ $# != 2 ]; then
-  echo "Usage: $0 [--per-utt] <data-dir> <num-to-split>"
+  echo "Usage: $0 [--per-utt|--per-reco] <data-dir> <num-to-split>"
```

Comment:
We'll discuss offline why you need this --per-reco option and whether we can get rid of it.